apiary.apib

```diff
@@ -2838,6 +2838,68 @@ Update a milestone.
 
 + Response 204 (application/json;charset=utf-8)
 
+## Materials [/materials]
+
+Materials are added to a project and they affect the profitability. This information is only available for superusers and project decision makers.
+
+*Required scopes: `projects`*
+
+### materials.list [GET /materials.list]
+
+Get a list of project materials.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + filter (object, optional)
+            + project_id: `082e6289-30c5-45ad-bcd0-190b02d21e81` (string, required)
```

Comment:
It's a bit weird to have a `.list` endpoint with a required filter in my opinion. This would make me think that this data structure should be part of the `products.info` endpoint similar to how invoice items are also part of an invoice.